recipes/gecode/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: gecode
+  version: "4.4.0"
+
+about:
+  home: http://www.gecode.org/
+  license: MIT
+  license_file: LICENSE
+  summary: Generic constraint development environment
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: gecode-4.4.0.tar.gz
+  url: "http://www.gecode.org/download/gecode-4.4.0.tar.gz"
+  sha256: b45783cc8d0d5dbbd3385a263a2199e6ad7f9a286e92607de81aa0c1105769cb
+requirements:
+  build: 
+    - gcc # [linux]
+    - libgcc # [linux]
+  run: 
+    - libgcc # [linux]
+
+test:
+  commands:
```

Comment:
Anything what we can test?
Maybe this: http://conda.pydata.org/docs/building/meta-yaml.html#test-files
Or this one? http://conda.pydata.org/docs/building/meta-yaml.html#run-test-script